firmware/controllers/actuators/idle_hardware.cpp

```diff
@@ -17,8 +17,8 @@
 #include "dc_motors.h"
 #if ! EFI_UNIT_TEST
 #include "stepper.h"
-static StepDirectionStepper iacStepperHw CCM_OPTIONAL;
-static DualHBridgeStepper iacHbridgeHw CCM_OPTIONAL;
+static StepDirectionStepper iacStepperHw;// CCM_OPTIONAL;
```

Comment:
plz kill dead code or add a comment, but not leave uncommented dead code

Reply:
done!